src/oss/langchain/middleware/custom.mdx

```diff
@@ -365,7 +365,17 @@ const loggingMiddleware = createMiddleware({
 
 ## Custom state schema
 
-Middleware can extend the agent's state with custom properties.
+Middleware can extend the agent's state with custom properties. This enables middleware to:
```

Comment:
Will readers know what agent state is at this point? (is there a cross-link that could be added if not)